src/components/ContentNode/Aside.vue

```diff
@@ -77,18 +60,41 @@ $aside-kinds: deprecated, experiment, important, note, tip, warning;
     $aside-width-left-border);
   padding: rem(16px);
 
+  .label {
+    @include font-styles(documentation-aside-name);
+
+    & + * {
+      margin-top: var(--spacing-stacked-margin-small);
+    }
+  }
+
   @each $kind in $aside-kinds {
     &.#{$kind} {
       background-color: var(--color-aside-#{$kind}-background);
       border-color: var(--color-aside-#{$kind}-border);
       // Anti-aliasing corners
       box-shadow: 0 0 $aside-width-border 0px var(--color-aside-#{$kind}-border) inset,
         0 0 $aside-width-border 0px var(--color-aside-#{$kind}-border);
+
+      .label {
+        color: var(--color-aside-#{$kind});
+      }
     }
   }
+}
 
-  .label {
-    @include font-styles(documentation-aside-name);
+// Apply a more compact styling for asides within a `@Steps` directive
+.steps {
```

Comment:
Should we do this from within the `Step.vue` or `SectionStep.vue` components? I am fine keeping it here, just thinking out loud.

Reply:
No preference, but I moved it 5c15fb3ae1d59826363b79167716dbf5f23020d4